src/stats/common.ts

```diff
@@ -137,6 +137,18 @@ export enum State {
   FALL_BACKWARD = 0x1f,
   GRAB = 0xd4,
   CLIFF_CATCH = 0xfc,
+  DAMAGE_FALL = 0x26,
+
+  // Command Grabs
+  BARREL_CANNON_WAIT = 0x153,
+  COMMAND_GRAB_RANGE1_START = 0x10a,
+  COMMAND_GRAB_RANGE1_END = 0x130,
+
+  COMMAND_GRAB_RANGE2_START = 0x147,
+  COMMAND_GRAB_RANGE2_END = 0x153,
```

Comment:
As Nikki mentioned already, why set the end range to `0x153` and then explicitly ignore `0x153`? Why not just set the end range to `0x152`?

```suggestion
  COMMAND_GRAB_RANGE2_END = 0x152,
```